### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by Philippus**

### CODE REVIEW

File: akka-docs/src/main/paradox/scala/io-tcp.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ to the network socket.
 
 These write back-pressure models (with the exception of the second which is rather specialised) are
 demonstrated in complete examples below. The full and contiguous source is
-available @extref[on GitHub](github:akka-docs/rst/scala/code/docs/io/EchoServer.scala).
+available @extref[on GitHub](github:akka-docs/src/test/scala/docs/io/EchoServer.scala).
```
Comment (ktoso) [inline akka-docs/src/main/paradox/scala/io-tcp.md]: thanks, good catch

**Review by ktoso — APPROVED** [review]
LGTM Thanks, nice cleanups :)

**Comment by ktoso** [comment]
OK TO TEST

**Review by johanandren — COMMENTED** [review]
LGTM

**Pull request merged**